### PR TITLE
Update CentralizedMockSBI.java remove and stop

### PR DESCRIPTION
**Pull request by neeharikatech**

### CODE REVIEW

**Review by lsivanand — COMMENTED** [review]
We are removing from map first and triggering stop.. to avoid spurious behavior if anything failed during Stop()

File: MockMDS/src/main/java/io/mosip/mock/sbi/test/CentralizedMockSBI.java [hunk 1]
```diff
@@ -46,8 +46,9 @@ public static void stopSBI(String context) {
         try {
             if(localStore.containsKey(context)) {
                 localStore.get(context).setStopped(true);
+                 localStore.remove(context);
```
Comment (gsasikumar) [inline MockMDS/src/main/java/io/mosip/mock/sbi/test/CentralizedMockSBI.java:49]: once you remove how can you get?

File: MockMDS/src/main/java/io/mosip/mock/sbi/test/CentralizedMockSBI.java [hunk 2]
```diff
@@ -60,8 +61,9 @@ public static void stopAllSBI() {
         for(String context : localStore.keySet()) {
             try {
                 localStore.get(context).setStopped(true);
+                 localStore.remove(context);
```
Comment (gsasikumar) [inline MockMDS/src/main/java/io/mosip/mock/sbi/test/CentralizedMockSBI.java:64]: same as above

**Review by gsasikumar — CHANGES_REQUESTED** [review]

**Comment by neeharikatech** [comment]
this PR is not required

**Pull request closed without merging**